docs/review_process.md

```diff
@@ -0,0 +1,136 @@
+Keep attention to the error messages from the bot, and address all the build failures.
+Bot tries to provide all the helpful information needed to understand and reproduce an issue, such as:
+
+- the profile that failed (in other words, the configuration - architecture, operation system, compiler, etc.)
+- failed command line (it might have failed on early stages - e.g. recipe syntax errors, or later stages, like build or test)
+- logs (contains actual output of the build process - not that some important logs like `configure.log` or `CMakeError.log` are not captured, it's just stdout/stderr)
+
+If you struggle to fix build errors yourself, you always may ask for the help, either by tagging individual users, or by adding label [Help Wanted](https://github.com/conan-io/conan-center-index/pulls?q=is%3Aopen+is%3Apr+label%3A%22help+wanted%22).
+
+### Unexpected error
+
+Sometimes, build fails with `Unexecpted error` message. This indicates an infrastracture problem, and usually it's unrelated to the changes within PR itself.
+Keep in mind CCI is still an early access (consider it beta), and there are still some instabilities. Especially, as we're using lots of various services,
+which might be inaccessible (GitHub API, docker hub, etc.) and may result in intermittent failures.
+So, what to do once `Unexecpted error` was encountered? You may consider rerunning the build, there are several ways to do it:
+
+- do an empty commit: `git commit --allow-empty`
+- update the last commit: `git commit --amend --no-edit && git push -f`
+- close pull request, wait 15 seconds, re-open pull request
+
+sometimes it's needed to be done several times.
+If an `Unexpected error` persists, you probably may assign an [Infrastructure](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3ADocs+label%3Ainfrastructure) label and tag [@jgsogo](https://github.com/jgsogo) and [@danimtb](https://github.com/danimtb) asking for the help with CI.
+Alternatively, just [open a new issue](https://github.com/conan-io/conan-center-index/issues/new/choose).
+
+## avoiding conflicts
+
+It's recommended to rebase your changes on top of the master branch to avoid conflicts.
+Right now, neither GitHub itself nor conan-center-bot notify about merge conflict, so it's contributor responsibility to periodically check for the conflicts.
+Obviously, PRs that have merge conflicts are never merged, and all the conflicts has to be resolved first.
+
+## draft
+
+Draft pull requests are also never merged. The same applies for `WIP` keyword (stands for `Work in Progress`) in the title - GitHub considers them drafts as well.
+Once you're done with your changes, remember to change from draft and remove WIP keyword from the title.
+
+## getting PR reviewed
+
+Each PR should be reviewed by several reviewers before it could be merged. It cannot be just reviews from random people, we have two categories of the reviewers:
```

Comment:
```suggestion
Each PR must be reviewed by several reviewers before it will be merged. It cannot be just reviews from random people, we have two categories of reviewers:
```